cincinnati/src/lib.rs

```diff
@@ -662,6 +662,8 @@ mod tests {
 
     use super::*;
 
+    type Fallible<T> = Result<T, Box<std::error::Error>>;
```

Comment:
I'd rather not squat `failure::Fallible` name, unless required. Can we either use that one directly or name it in our own way (e.g. `TestResult`)?